application/language/dutch/kalkun_lang.php

```diff
@@ -412,3 +410,9 @@
 $lang['Group'] = 'Group';
 $lang['Delete the original message (prevents duplicates).'] = 'Delete the original message (prevents duplicates).';
 $lang['Delete permanently'] = 'Delete permanently';
+$lang['Writable'] = 'Writable';
+$lang['Read-only'] = 'Read-only';
+$lang['No users in the database.'] = 'No users in the database.';
```

Comment:
Can we fit some existing translation here? Ex 'Users is empty'. It won't be entirely correct, but better than having English

Reply:
done